translations/ar/LC_MESSAGES/messages.po

```diff
@@ -36,10 +35,6 @@ msgstr "البرنامج الخاص بك غير مكتمل. عليك أن تست
 msgid "Incomplete"
 msgstr "أوه! لقد نسيت أن تكمل جزءاً من البرنامج! في السطر رقم {line_number}، عليك أن تكمل كتابة الكود بعد {incomplete_command}."
 
-#, fuzzy
```

Comment:
Hi @akseron!

Your PR works great but I am not sure why this string is removed? I think this is stemming from one of your earlier PR's and should stay in?